desktop/main/walletFile.ts

```diff
@@ -270,6 +270,55 @@ export const copyWalletFile = async (filePath: string, outputDir: string) => {
   return newFilePath;
 };
 
+export const isWalletFileValid = async (filePath: string) => {
```

Comment:
This entire function should not persist in such a form.
Instead, please implement `isWalletMeta` and `isWalletFile` guards within `shared/types/guards.ts` module. And use it.

Pay attention, that they should not do any I/O or side effects.